src/super_gradients/common/crash_handler/crash_tips.py

```diff
@@ -0,0 +1,209 @@
+            :param exc_traceback:   Traceback
+
+            :return:                Solution
+        """
+        raise NotImplementedError
+
+    @classmethod
+    def _get_tip(cls, exc_type: type, exc_value: Exception, exc_traceback: TracebackType) -> str:
+        """
+        Provide a customized tip for the exception, combining explanation and solution.
+
+        Beside the class, the input params are as returned by sys.exc_info():
+            :param cls:             Class inheriting from CrashTip
+            :param exc_type:        Type of exception
+            :param exc_value:       Exception
+            :param exc_traceback:   Traceback
+
+            :return:                Tip
+        """
+        explanation = indent_string(cls._get_explanation(exc_type, exc_value, exc_traceback), indent_size=4)
+        solution = indent_string(cls._get_solution(exc_type, exc_value, exc_traceback), indent_size=4)
+
+        msg = (
+            "- What went wrong:\n"
+            f"{explanation}\n"
+            "- How to solve it:\n"
+            f"{solution}\n"
+            f"- If the proposed solution did not help, feel free to contact the SuperGradient team or to open a ticket: "
+            f"https://github.com/Deci-AI/super-gradients/issues/new/choose"
+        )
+        return msg
+
+    @classmethod
+    def get_message(cls, exc_type: type, exc_value: Exception, exc_traceback: TracebackType) -> Union[None, str]:
+        """
+        Wrap the tip in a nice message.
+
+        Beside the class, the input params are as returned by sys.exc_info():
+            :param cls:             Class inheriting from CrashTip
+            :param exc_type:        Type of exception
+            :param exc_value:       Exception
+            :param exc_traceback:   Traceback
+
+            :return:                Tip
+        """
+        try:
+            tip = indent_string(cls._get_tip(exc_type, exc_value, exc_traceback), indent_size=4)
+            message = (
+                "******************************************************************************************************************************\n"
+                "Something went wrong!\n"
+                "Here is our guess:\n"
+                f"{tip}\n"
+                "see the trace above...\n"
+                "******************************************************************************************************************************\n"
+            )
+            return "\n" + message
+        except Exception:
+            # It is important that the crash tip does not crash itself, because it is called atexit!
+            # Otherwise, the user would get a crash on top of another crash and this would be extremly confusing
+            return None
+
+
+class TorchCudaMissingTip(CrashTip):
+    @classmethod
+    def is_relevant(cls, exc_type: type, exc_value: Exception, exc_traceback: TracebackType) -> bool:
+        pattern = "symbol cublasLtHSHMatmulAlgoInit version"
```

Comment:
This error may actually indicate CUDA libraries version conflict (When Torchvision & Torch are installed for different CUDA versions) and the absence of CUDA support in PyTorch.

Reply:
Thanks, I'll check that

Reply:
I improved the tip accordingle: a1c3a16425b8fbaf66b3b65883751d3ec1390e8a